### PR TITLE
Remove CSS modules from components/Menu

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**
Extensions sidebar updated to match Layout styles

Screen Shot 2021-09-21 at 3 58 09 pm

**Description**
extensions sidebar was importing from `components/Menu` 🤷 not sure why 🔨 

Related to larger style approach refactor #1691

### CODE REVIEW

File: packages/studio-base/src/components/Menu/Item.tsx [hunk 1]
```diff
@@ -72,35 +132,40 @@ const Item = (props: ItemProps): JSX.Element => {
...
+        <span className={classes.icon}>
+          <Icon size={iconSize}>{icon}</Icon>
         </span>
       )}
-      <SContentWrapper>
+      <div className={classes.contentWrapper}>
         {children}
         {checked && !isDropdown && (
-          <Icon {...{ [iconSize]: true }} style={{ marginLeft: "5px" }}>
+          <Icon size={iconSize} style={{ marginLeft: "5px" }}>
```
Comment (2metres) [inline packages/studio-base/src/components/Menu/Item.tsx:161]: Missed these changes on #1785 due to prop spreading

**Comment by defunctzombie** [comment]
👍

**Comment by jtbandes** [comment]
> extensions sidebar was importing from `components/Menu` 🤷 not sure why 🔨

Hmmm it's possible it was an accidental auto-import provided by VS Code...anyway, good find

**Comment by jtbandes** [comment]
Fixes #1778? Or only part of it if this doesn't change the extension details view?

**Comment by 2metres** [comment]
> Fixes #1778? Or only part of it if this doesn't change the extension details view?

Only the Extensions Sidebar not the details view

**Pull request merged**